### PR TITLE
Fixes for chrome environment

### PR DESCRIPTION
**Pull request by SerhiiKun**

### CODE REVIEW

File: src/sdk.js [hunk 1]
```diff
@@ -138,7 +158,8 @@ class EventsSDK {
   _onConnect() {
     this._initReconnectDelays();
     this.connected = true;
-    this.Logger.log(eventTypes.CONNECT, this.reconnectOptions);
+    //log(eventTypes.CONNECT, this.reconnectOptions);
```
Comment (Regani) [inline src/sdk.js:161]: Let's remove this

File: src/sdk.js [hunk 2]
```diff
@@ -182,13 +203,17 @@ class EventsSDK {
       this.socket.io.reconnectionDelayMax(newDelay);
     }
     this.reconnectOptions.retryCount++;
-    this.Logger.log(eventTypes.RECONNECT_ATTEMPT, this.reconnectOptions)
+    this.log('INFO', eventTypes.RECONNECT_ATTEMPT, this.reconnectOptions)
```
Comment (Regani) [inline src/sdk.js:206]: Let's create constants for 'INFO'm
 'ERROR' and 'WARN'

File: src/sdk.js [hunk 3]
```diff
@@ -182,13 +203,17 @@ class EventsSDK {
...
   }
 
   _onDisconnect(reason) {
     this.connected = false;
-    this.Logger.log(eventTypes.DISCONNECT, reason);
-    this._connect('next', true)
+    //this.Logger.log(eventTypes.DISCONNECT, reason);
```
Comment (Regani) [inline src/sdk.js:211]: -

File: src/sdk.js [hunk 4]
```diff
@@ -197,7 +222,8 @@ class EventsSDK {
       return
     }
     if (data && this.connected) {
-      this.Logger.log(eventTypes.KEEP_ALIVE_RESPONSE);
+      //this.Logger.log(eventTypes.KEEP_ALIVE_RESPONSE);
```
Comment (Regani) [inline src/sdk.js:225]: -

**Review by Regani — CHANGES_REQUESTED** [review]

File: src/sdk.js (hunk 1 above)
Comment (SerhiiKun) [reply src/sdk.js]: Done

**Review by Regani — APPROVED** [review]

**Pull request merged**